src/app/components/Search/index.tsx

```diff
@@ -113,7 +127,20 @@ const SearchCmp: FC<SearchProps> = ({ scope, variant, disabled, onFocusChange: o
   const onFormSubmit = (e?: FormEvent) => {
     e?.preventDefault()
     if (value) {
```

Comment:
(This was previously the only validation step, now there's three. It doesn't look easily mergeable into `if (value && !tooShort && !privacyProblem) {`, so I would usually do: `if (!value) return` and ` if (tooShort) return` and `if (privacyProblem) return` )

Reply:
Up, refactored that way.